speaking-portal-project/src/phonemeFactory/rhubarb.ts

```diff
@@ -37,11 +37,17 @@ export async function rhubarbProcessor(selected_language: string, audio_file_nam
     const rhubarbProc = spawnSync('./rhubarb', args)
 
     if (rhubarbProc.stderr) {
-        //throw Error(`${rhubarbProc.stderr}`)
+        // Rhubarb returns this as an error but it's just a status message, so do NOT throw this as an error
+        if(!rhubarbProc.stderr.includes(`Generating lip sync data for ${audio_file_name}.`)){
```

Comment:
simple fix! Interesting that rhubarb & ffmpeg both throw their outputs as errors